### ISSUE TITLE
Can I use a Kodi Addon that ads IMDb ratings?

### ISSUE THREAD

**Issue by viking2** [croneter/PlexKodiConnect#1290]
I would like to use a Kodi Addon that ads IMDb ratings:

https://forum.kodi.tv/showthread.php?tid=316342#

Would those ratings be overwritten when syncing with Plex database?

**Comment by croneter** [comment]
Yes. I think you should be able to configure PLEX to use IMDB ratings - problem solved.

**Comment by viking2** [comment]
Plex (with or without Addons) no longer support IMDb ratings for TV-shows. That is why I need to use a Kodi Addon.
Would those ratings be overwritten when syncing with Plex database?

**Comment by croneter** [comment]
Pkc would overwrite your IMDB ratings at some point, if an item changes (e.g. changed Plex metadata). Go ahead and try to overwrite the ratings with IMDB. You cannot really break anything

**Comment by viking2** [comment]
I created a separate profile w/o PKC, and I see the IMDb ratings for my TV-Shows in Kodi.
Note that both Native Kodi and the Kodi Add-on gets the IMDb ratings although, for some reasons, they are different.

However, with the PKC profile, I don't see the ratings :-( . 
(I only see the ratings that are available through Plex)

I guess that there is no workaround to see the Kodi IMDb ratings when I use PKC?

**Comment by croneter** [comment]
Pkc syncs whatever metadata you have on the Plex side.please check with the Plex manuals how to add IMDB. See e.g. https://forums.plex.tv/t/put-imdb-and-or-rotten-tomatoes-rating-on-my-plex/478039

**Comment by viking2** [comment]
It only works for Movies. Not TV-Shows.

**Comment by croneter** [comment]
https://forums.plex.tv/t/plex-agents-and-ratings-for-dummies/549500#:~:text=You%20cannot%20get%20IMDb%20ratings,particular%20tv%20show%20in%20question.

You cannot get IMDb ratings for TV shows.
All TV-related metadata come from either TheTVDB or TheMovieDB, depending on which of these two agents was used to match the particular tv show in question.

**Comment by viking2** [comment]
Yes, I am aware that Plex agents cannot get ratings for TV-shows. That is what I explained above.

As I also explained, that is why I use the Kodi add-on ("Light IMDb Ratings Update"):
https://forum.kodi.tv/showthread.php?tid=316342
It scrapes the TV-Shows data from the IMDb site.

I have tested it and it works great in Kodi (**using a profile w/o PKC**)!
Unfortunately, it doesn't work when I use a profile with PKC.
